### DISCUSSION TITLE
Do you support OAuth 2.0 Client Credentials Flow?

### DISCUSSION THREAD

**Discussion by dwaltz** [jsforce/jsforce#1497] [Q&A] [answered]
[Documentation](https://help.salesforce.com/s/articleView?id=sf.connected_app_client_credentials_setup.htm&type=5)

**Comment by cristiand391** [comment, marked as answer]
**EDIT**:
See: https://jsforce.github.io/document/#client-credentials-flow-oauth-2-0-client-credentials-flow-

-------------------

Hey, sorry for the late reply!

yes, the client credentials flow is supported (in v3), we have a PR to add an example in docs:
https://github.com/jsforce/jsforce.github.io/pull/17

https://github.com/jsforce/jsforce/issues/1371#issuecomment-2130412125
